doc/source/faq.rst

```diff
@@ -125,9 +78,45 @@ What is UFO?
 UFO is a general purpose image processing framework, optimized for heterogeneous
 compute systems and streams of data. Arbitrary data processing tasks are plugged
 together to form larger processing pipelines. These pipelines are then mapped to
-the hardware resources available at run-time, i.e. both multiple GPUs and CPUs.
+the hardware resources available at run-time, i.e. all available multiple GPUs
+and CPUs.
 
 One specific use case that has been integrated into the TomoPy is fast
 reconstruction using the filtered backprojection and direct Fourier inversion
 methods although others for pre- and post-processing might be added in the
 future.
+
+
+Which platforms are supported?
+==============================
+
+TomoPy supports Windows, Linux, and macOS. The ASTRA toolbox supports Linux
+and Windows. Project UFO supports Linux and macOS.
+
+
+Do TomoPy, ASTRA, and UFO support all GPUs?
+===========================================
+
+The GPU algorithms in TomoPy and the ASTRA toolbox are all implemented using
+nVidia's CUDA. As a result, only nVidia CUDA­ enabled GPUs can be used to run
+them. UFO uses OpenCL, so it supports both AMD and nVidia OpenCL compatible
+GPUs.
+
+
+Can I run this on a HPC cluster?
+================================
+
+Maybe. In their default installation packages, TomoPy and the ASTRA toolbox are
+limited to running on a single multi-core and multi-GPU machine. Both toolboxes
+can be run on a HPC cluster through parallelization using MPI, but since
+installation and running on a HPC cluster is often cluster­ specific, the
+default installation packages do not include these capabilities.
+
+As such, the integrated packages that is presented in the manuscript currently
+does not support running on a HPC cluster. Note that the ASTRA toolbox provides
+a separate MPI­ enabled package for use on a HPC cluster. We refer to
+:cite:`Bicer:15` for more details about TomoPy's planned HPC implementation. It
+is a MapReduce type MPI implementation layer, which was successfully used on
+many clusters,  i.e. Stampede, Cori, Mira. There are plans to allow user access
+to TomoPy on a HPC cluster (e.g. through a client or web­portal), but these
+projects will take some time before they are being matured for user’s use.
```

Comment:
Answer the question of why we don't have pip install here?

Reply:
Done.